src/com/phonebookservice/server/ContactConverter.java

```diff
@@ -0,0 +1,77 @@
+package com.phonebookservice.server;
+
+import java.util.Arrays;
+
+import com.phonebookservice.model.Address;
+import com.phonebookservice.model.Contact;
+import com.phonebookservice.model.PhoneNumber;
+import com.phonebookservice.model.PhoneNumber.PhoneLabel;
+import com.phonebookservice.util.CollectionUtility;
+
+public final class ContactConverter {
+    public static final String SPLITTER = ",";
+
+    private ContactConverter() {
+
+    }
+
+    /**
+     * map string to contact.
+     *
+     * @param splitLine the line.
+     *
+     * @return contact.
+     */
+    public static Contact toContact(final String[] splitLine) {
+        final Address address = Address.builder()
```

Comment:
I suggest this method takes a string and split it here.

We are doing this because u have another method convert contact to a string

So this class is a converter between string and contact object. And vice versa.

This is the only place that understand this conversion